### DISCUSSION TITLE
Cypress tests crashing error

### DISCUSSION THREAD

**Discussion by Shubham1406220** [cypress-io/cypress#26093] [Questions and Help]
Getting this error when running cypress on jenkins
`Gdk-Message: 05:43:30.722: Cypress: Fatal IO error 0 (Success) on X server :99.`

**Comment by bilarallen** [comment]
Hi @Shubham1406220 are you using cypress docker image? in the docker image there is xvfb included if you're not using docker image try to install `xvfb` and in your command in running the test add this command `xvfb-run -a npx cypress run ...` add this command whether you use docker image or not

**Reply by Shubham1406220** [reply]
hey @bilarallen , we are not running in a container. It is installed on a server. Will try the one you suggested when we face this isue again, since the issue is very intermittent. Thanks
